xee/ext.py

```diff
@@ -634,7 +634,7 @@ def reduce_bands(x, acc):
     return target_image
 
   def _raw_indexing_method(
-      self, key: tuple[Union[int, slice], ...]
+      self, key: tuple[Union[int, slice], ...], executor_kwargs: Optional[dict] = None
```

Comment:
This extra argument is named well, but I think it should be set in the constructor.